Controller/SubscriptionWallController.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/*
+* This file is part of the Integrated package.
+*
+* (c) e-Active B.V. <integrated@e-active.nl>
+*
+* For the full copyright and license information, please view the LICENSE
+* file that was distributed with this source code.
+*/
+
+namespace Integrated\Bundle\SubscriptionBundle\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+
+/**
+ * @author Jacob de Graaf <jacob.de.graaf@windesheim.nl>
+ * @author Albert Bakker <albert-david.bakker@windesheim.nl>
+ */
+class SubscriptionWallController extends Controller
+{
+    /**
+     * Lists the walls
+     *
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    public function indexAction()
+    {
+        $walls = $this->getDoctrine()
+            ->getRepository('Integrated\Bundle\SubscriptionBundle\Model\SubscriptionWall')
+            ->findAll();
+        if (!$walls) {
```

Comment:
No need for this if statement. It is better not to return multiple different views. In most cases it should be sufficient to have just one $this->render function at the end.